filter/etc/e-smith/templates/etc/httpd/admin-conf.d/rspamd.conf/10base

```diff
@@ -5,6 +5,7 @@
     my $confdb = esmith::ConfigDB->open;
     my $rspamd = $confdb->get('rspamd') or die "No rspamd db entry found\n";
     my $alias = $rspamd->prop('alias') || "";
+    my $user ="admin admin\@$DomainName administrator administrator\@$DomainName";
```

Comment:
We should honor the settings from `admins/admin` `admins/group` props in configuration DB

Reply:
ok